frontend/app-preview/src/views/LandingPage.module.css

```
@@ -1,5 +1,9 @@
+:root {
+  --previewLimitationsInfo-height: 0px;  /* Default height which will be overriden */
+}
+
 .header {
-  font-family: Inter, 'San Fransisco', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-family: var(--font-family);
```

Vi har en global studio variable for dette nå. 😊 
```suggestion
  font-family: var(--studio-font-family);
```